### ISSUE TITLE
Can't install it on ubuntu

### ISSUE THREAD

**Issue by mohsentaleb** [andrewharvey/geojson-polygon-labels#7]
Running `npm install -g geojson-polygon-labels` fails with the following error message in `Ubuntu 18.04 LTS`:
```
node-pre-gyp ERR! Tried to download(403): https://mapbox-node-binary.s3.amazonaws.com/zipfile/v0.5.11/node-v57-linux-x64.tar.gz
node-pre-gyp ERR! Pre-built binaries not found for zipfile@0.5.11 and node@8.10.0 (node-v57 ABI) (falling back to source compile with node-gyp)
```

`node -v // v8.10.0 ` 
`npm -v // 3.5.2`

**Comment by andrewharvey** [comment]
I guess this is coming from the dependency https://github.com/mapbox/stream-features-from-file, could you try installing `@mapbox/stream-features-from-file`, if that fails too, then it's the upstream issue. An older version doesn't use stream-features-from-file, so that might work until this is fixed.

**Comment by jfrankl** [comment]
@andrewharvey I'm not able to install this package using `node v10.15.1` and `npm v3.8.0`. Seems like the same issue with `@mapbox/stream-features-from-file` issue @mohsentaleb was experiencing.

I wanted to see if you'd be open to a PR that addresses this issue. I suspect that I would remove the dependency `@mapbox/stream-features-from-file` and switch to using https://github.com/node-geojson/geojson-stream, which seems like it is maintained and has similar functionality. I'm open to other solutions.

This library has been very helpful to me on a couple of projects, where I've switched to older versions of node to use it. I'd love to be able to pitch in to get it working on the latest version of node. Please let me know what you think and what direction you'd want to see this go.

**Comment by andrewharvey** [comment]
Hi @jfrankl Yes I would like to move away from `@mapbox/stream-features-from-file` since it's a heavy dependency and includes Mapnik which can be a pain to install.

I tried to swap to geojson-stream in my branch https://github.com/andrewharvey/geojson-polygon-labels/compare/geojson-stream however I ran into a problem when one incoming feature was split into multiple output features when exploding multipolygons, I couldn't work out how to do that with geojson-stream :confused: 

So I'd gladly accept a PR moving to `geojson-stream` if you can get it working. I don't mind if you either start fresh or base it on my working branch.

Thank you.

**Comment by jfrankl** [comment]
@andrewharvey thanks for your reply and for pointing me to your `geojson-stream` branch. That will will save me a lot of time.

**Comment by andrewharvey** [comment]
This should be fixed now with `geojson-stream` just released in v1.3.0.
